src/components/Light/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import onImg from "./img/light-on.svg"
+import offImg from "./img/light-off.svg"
+
+const Light = ({name, state}) => {
+    
+    const [localState, setLocalState] = useState (state)
+
+
+    const handleClick = () => {
+        setLocalState (localState === 'on'? 'off' : 'on') 
+        
+        // if (localState === 'on') {
+        //     setLocalState ('off')
+        // } else {
+        //     setLocalState ('on')
+        // }
+    }
+    
+    return(
+        <div onClick={handleClick} className = 'light'>
+            <div className="light__icon">
+				<img src={localState === 'on'? offImg : onImg}/>
```

Comment:
máš to opačně :))
![image](https://user-images.githubusercontent.com/8430123/141334112-10e9530b-b6cc-4b07-8bed-ab374a0db595.png)
![image](https://user-images.githubusercontent.com/8430123/141334097-8355b813-dfc6-49d1-b880-74a957bcee0e.png)